keep-alive/src/test/java/KeepAliveGattTest.kt

```diff
@@ -350,6 +354,148 @@ class KeepAliveGattTest {
         }
     }
 
+    @Test
+    fun `When 'connect' occurs, the onConnect lambda is called once`() = runBlocking {
```

Comment:
The name of this function is maybe outdated?

```suggestion
    fun `When 'connect' occurs, the onEventAction is called with Connected event`() = runBlocking {
```